src/theory/builtin/theory_builtin_rewriter.h

```diff
@@ -46,7 +46,31 @@ class TheoryBuiltinRewriter {
   }
 
   static inline RewriteResponse postRewrite(TNode node) {
-    return doRewrite(node);
+    if( node.getKind()==kind::LAMBDA ){
```

Comment:
This is kind of big for an inlined function.  Maybe move the whole thing to the cpp file.  Or at least the LAMBDA case.